packages/trader/src/Modules/Trading/Components/Form/ContractType/contract-type-dialog.jsx

```diff
@@ -52,9 +52,18 @@ const ContractTypeDialog = ({
 );
 
 ContractTypeDialog.propTypes = {
+    categories: PropTypes.array,
     children: PropTypes.element,
+    is_info_dialog_open: PropTypes.bool,
+    is_open: PropTypes.bool,
+    item: PropTypes.object,
+    list: PropTypes.oneOfType([PropTypes.array, PropTypes.object]),
+    onBackButtonClick: PropTypes.func,
+    onCategoryClick: PropTypes.func,
+    onChangeInput: PropTypes.func,
     onClose: PropTypes.func,
     open: PropTypes.bool,
+    selected: PropTypes.bool,
```

Comment:
Shouldn't it be `is_selected`?

Reply:
They defined the property name like this and it is `selected`